ansible/roles/aodh/tasks/config.yml

```diff
@@ -43,6 +43,20 @@
   notify:
     - "Restart {{ item.key }} container"
 
+- name: Copying over extra CA certificates
+  become: true
+  copy:
+    src: "{{ node_config }}/certificates/ca/"
```

Comment:
@brtknr do we need to teach Kayobe about this `ca` folder?

Reply:
This has the required commit: https://github.com/stackhpc/kayobe/tree/stackhpc/train

Reply:
Thanks!